tests/tests.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require '../functions.php';
+
+class Tests extends TestCase
+{
+    public function testDisplayChairs_success1()
+    {
+        $test_array = [[
+            'chairname' => 'highchair',
+            'designer' => 'gary',
+            'designyear' => '2001'
+        ]];
+        $result = displayChairs($test_array);
+        $expected = '<h3>highchair' . '</h3>' . '<span>Designer: gary' . '</span><br/>' . '<span>Year: 2001' . '</span><br/>';
+        $this->assertEquals($result, $expected);
+    }
```

Comment:
How about a malformed test? What happens if the displayChairs() functions is given a boolean for example?

Also a failure test could go here, what happens if displayChairs is given an empty array? IE no results from the database?